lib/index.js

```diff
@@ -36,6 +36,7 @@ GruntfileEditor.prototype.insertConfig = function (name, config) {
   var gruntConfig = this.gruntfile.callExpression('grunt.initConfig').arguments.at(0);
   // Select nested paths
   var targetConfig = name.split('.').reduce(function (nestedConfig, key) {
+    nestedConfig.node.range || nestedConfig.value('{}') //create empty object when value is undefined
```

Comment:
Make this a `if` statement.

Move the comment to the line above and add a space between `//` and text.